apps/web/templates/application.html.erb

```diff
@@ -5,6 +5,12 @@
     <%= favicon %>
   </head>
   <body>
+    <% flash[:errors]&.each do |error_message| %>
```

Comment:
should this logic belong in `views/application_layout.rb` with tests?

Reply:
How you picturing that looking?

Reply:
I was thinking a method which encapsulates the logic that renders the flash errors as HTML. Probably not worthwhile since we're only doing this in one place anyway. Happy to proceed without this